mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -338,6 +337,327 @@ def _load_mesh(self):  # {{{
+
+                annualClimatology['avgVertVelocityTop'] = \
+                    annualClimatology['timeMonthly_avg_vertVelocityTop'] + \
+                    annualClimatology['timeMonthly_avg_vertGMBolusVelocityTop']
+            else:
+                # rename some variables for convenience
+                annualClimatology = annualClimatology.rename(
+                    {'timeMonthly_avg_normalVelocity': 'avgNormalVelocity',
+                     'timeMonthly_avg_vertVelocityTop': 'avgVertVelocityTop'})
+
+            # Convert to numpy arrays
+            # (can result in a memory error for large array size)
+            horizontalVel = annualClimatology.avgNormalVelocity.values
+            verticalVel = annualClimatology.avgVertVelocityTop.values
+            velArea = verticalVel * areaCell[:, np.newaxis]
+
+            # Create dictionary for MOC climatology (NB: need this form
+            # in order to convert it to xarray dataset later in the script)
+            self.depth = refTopDepth
+            self.lat = {}
+            self.moc = {}
+            for region in self.regionNames:
+                self.logger.info('   Compute {} MOC...'.format(region))
+                if region == 'Global':
+                    transportZ = np.zeros(nVertLevels)
+                else:
+                    maxEdgesInTransect = \
+                        self.dictRegion[region]['maxEdgesInTransect']
+                    transectEdgeGlobalIDs = \
+                        self.dictRegion[region]['transectEdgeGlobalIDs']
+                    transectEdgeMaskSigns = \
+                        self.dictRegion[region]['transectEdgeMaskSigns']
+                    transportZ = self._compute_transport(maxEdgesInTransect,
+                                                         transectEdgeGlobalIDs,
+                                                         transectEdgeMaskSigns,
+                                                         nVertLevels, dvEdge,
+                                                         refLayerThickness,
+                                                         horizontalVel)
+
+                regionCellMask = self.dictRegion[region]['cellMask']
+                latBinSize = \
+                    config.getExpression(self.sectionName,
+                                         'latBinSize{}'.format(region))
+                if region == 'Global':
+                    latBins = np.arange(-90.0, 90.1, latBinSize)
+                else:
+                    indRegion = self.dictRegion[region]['indices']
+                    latBins = latCell[indRegion]
+                    latBins = np.arange(np.amin(latBins),
+                                        np.amax(latBins)+latBinSize,
+                                        latBinSize)
+                mocTop = self._compute_moc(latBins, nVertLevels, latCell,
+                                           regionCellMask, transportZ, velArea)
```

Comment:
The following 3 lines is the bit to compute the MOC at 26.5. I need to put this into a region loop, keeping in mind that MOC online has two separate arrays: one for the Global bit and one for the regional bit. At the moment, the latter is for the Atlantic, if `indRegion=0`.